addons/vehicleTrenches/XEH_postInit.sqf

```diff
@@ -0,0 +1,5 @@
+#include "script_component.hpp"
+
+private _id = [QGVAR(addBuildHandler), { _this call FUNC(addBuildHandler); }] call CBA_fnc_addEventHandler;
+
+TRACE_1(Registered 'addBuildHandler',_id);
```

Comment:
... does this work as-is? I'm shocked this didn't throw an error when building or something.
```suggestion
TRACE_1("Registered addBuildHandler",_id);
```

Reply:
... yeah I'm amazed that's actually been working at all lmao